libs/sdk-ui-dashboard/src/converters/filterConverters.ts

```diff
@@ -92,17 +94,17 @@ export function filterContextDateFilterToDateFilter(
  *
  * @param filterContextItems - filter context items to get filters for
  * @param widget - widget to use to get dateDataSet for date filters
- * @internal
+ * @public
  */
-export function filterContextItemsToFiltersForWidget(
+export function filterContextItemsToDashboardFilters(
```

Comment:
Maybe we could also use a version that does not take widget, but a date attribute? Because the user "outside" of the component might want to use this not for some widget.

Reply:
Created mew functions taking dateDataSet ObjRef object instead of the widget. These are distinguished by name, either you can convert byWidget or byDateDataSet.